### PR TITLE
UCD files for 16.0 with 2024 dates

### PR DESCRIPTION
**Pull request by markusicu**
- some generated with the new copyright year in MakeUnicodeFiles.txt
- the rest edited manually

### CODE REVIEW

**Review by rickmcgowan — APPROVED** [review]
Looks to me like basically only file date changes.

**Comment by markusicu** [comment]
> Looks to me like basically only file date changes.

... and some versions from 15.1 to 16.0

**Comment by markusicu** [comment]
@Ken-Whistler do you see anything fishy here? I updated all copyright dates, regardless of whether contents had changed this year.

File: unicodetools/data/ucd/dev/BidiTest.txt [hunk 1]
```diff
@@ -66,7 +66,9 @@
 #   @Reorder:    3 1
 #
 # A data line has the following format:
+
 # <input> ; <bitset>
+
```
Comment (eggrobin) [inline unicodetools/data/ucd/dev/BidiTest.txt:71]: Blank lines in the headers should still start with `#`.
Also, why is this bundled with the date changes?

**Review by eggrobin — DISMISSED** [review]

File: unicodetools/data/ucd/dev/BidiTest.txt (hunk 1 above)
Comment (markusicu) [reply unicodetools/data/ucd/dev/BidiTest.txt:71]: This commit is the output (verbatim) from a MakeUnicodeFiles run after changing the copyright year to 2024.

Comment (markusicu) [reply unicodetools/data/ucd/dev/BidiTest.txt:71]: If the generator needs to be changed, can we do that in a separate PR please?

**Pull request merged**